src/scripts/hadolint.sh

```diff
@@ -2,6 +2,34 @@
 eval "$SCRIPT_UTILS"
 expand_env_vars_with_prefix "PARAM_"
 
+if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+
+Install_Hadolint() {
+  if uname -a | grep "Darwin"; then
+    export SYS_ENV_PLATFORM="Darwin"
```

Comment:
Are you using `SYS_ENV_PLATFORM` outside this script? If the answer is "no", I believe you don't to `export` the variable.